content/_index.md

```diff
@@ -62,4 +61,4 @@ When you're well aware of an ongoing problem, [silence its alerts][18]. When you
 [17]: /integrations/hipchat
 [18]: /monitors/downtimes
 [19]: /monitors/downtimes/
-[20]: /monitors/monitor_types/composite/
\ No newline at end of file
+[20]: /monitors/monitor_types/composite/
```

Comment:
My editor added a trailing newline here. Feel free to remove if it's inappropriate.